rcl/include/rcl/time.h

```diff
@@ -447,7 +447,7 @@ rcl_enable_ros_time_override(rcl_time_source_t * time_source);
 RCL_PUBLIC
 RCL_WARN_UNUSED
 rcl_ret_t
-rcl_disable_ros_time_override(rcl_time_source_t * time_source);
+rcl_ros_time_disable_override(rcl_time_source_t * time_source);
```

Comment:
Same thing for this one, revert it because `ros_time` is a concept not a type as I had originally thought.